### PR TITLE
Complete Task 7 (Steps 1 - 5)

### PR DESCRIPTION
**Pull request by rhoadesd94**
Step 1 -
Added the "Mark As Done" button per Step 1's directions.

Step 2 -
Incorporated "Mark As Done" click recognition per Step 2's directions.
I also refactored .getElementById method calls to .querySelector method calls.

Step 3 -
Included the task ID into the HTML template literal so that the "Mark As Done" button returns an HTML element with the same ID as the task that was marked as done.

Step 4 - 
Create a .getTaskById method in the Task Manager task that returns a task with a given ID (tested by manually typing _taskManager.getTaskById(0) into the console - note that the ID is passed into this method as a number not a string).

Step 5 -
Modify the functionality of the "Mark As Done" button's event handler to update the status of the task in question to 'DONE'.

*Note: I did not spend time on the stretch goals as I have other Canvas work I need to catch up on.

### CODE REVIEW

File: Task-6/js/index.js [hunk 1]
```diff
@@ -83,11 +83,26 @@ function validFormFieldInput(event) {
...
-  submitElement = document.getElementById('submit-form');
+  submitElement = document.querySelector('#submit-form');
   submitElement.addEventListener("click", validFormFieldInput);
+
+  
+  let renderList = document.querySelector('#render-task-list');
+  renderList.addEventListener("click", (event) => {
+    // console.log(event.target.classList);
+    if(event.target.classList.contains('done-button')) {
+      let parentTask = event.target.parentElement.parentElement.parentElement;
```
Comment (flx1derrick) [inline Task-6/js/index.js:100]: Everything works as expected, however on line 100 of index.js chaining .parentElement's to traverse the DOM could break our code if the structure of our card changes in the future -- adopting the .closest() method is an alternative approach ( for example if the button moves to a different node  ) 
My suggestion is this:
`100 let parentTask = event.target.closest('li')`

**Review by flx1derrick — CHANGES_REQUESTED** [review]
Everything works as expected, however on line 100 of index.js chaining .parentElement's to traverse the DOM could break our code if the structure of our card changes in the future -- adopting the .closest() method is an alternative approach ( for example if the button moves to a different node  ) 
My suggestion is this:
`100 let parentTask = event.target.closest('li')`

File: Task-6/js/index.js (hunk 1 above)
Comment (rhoadesd94) [reply Task-6/js/index.js]: Love that observation! Just committed the recommended implementation.

**Review by rhoadesd94 — COMMENTED** [review]
As mentioned in the reply, the latest commit implements the recommended code

**Review by flx1derrick — APPROVED** [review]
index.js line 100 resolved. merging.

**Pull request merged**